args4j/src/org/kohsuke/args4j/CmdLineParser.java

```diff
@@ -548,7 +548,7 @@ public void parseArgument(final String... args) throws CmdLineException {
      * by the @ sequences, split around the line breaks.
      * @throws CmdLineException 
      */
-    private String[] expandAtFiles(String args[]) throws CmdLineException {
+    protected String[] expandAtFiles(String args[]) throws CmdLineException {
         List<String> result = new ArrayList<String>();
```

Comment:
not clear why whole method needs to be overidable rather than just readAtFile. Since this method takes whole args list as parameter,  makes it too easy to do generally bad things.